example/index.js

```diff
@@ -143,6 +151,13 @@ async function init() {
 	ptRenderer.material.bgGradientTop.set( params.bgGradientTop );
 	ptRenderer.material.bgGradientBottom.set( params.bgGradientBottom );
 
+	temporalResolve = new TemporalResolve( ptRenderer, scene, activeCamera );
+	temporalResolve.temporalResolveMix = 0.9;
+	temporalResolve.clampRadius = 1;
+	temporalResolve.newSamplesSmoothing = 0.5;
+	temporalResolve.newSamplesCorrection = 0.75;
+	temporalResolve.weightTransform = 0;
```

Comment:
It may be best to initialize these parameters from the `params` object - otherwise if we change the demo defaults we have to remember to do it in two places.